models/workspacemanager/workspace_provider.py

```diff
@@ -2,7 +2,7 @@
 
 class WorkspaceProvider:
     @abc.abstractmethod
-    def getWorkspaceNames(self):
+    def mtd_getObjectNames(self):
```

Comment:
Don't call it `mtd`. This supposes the use of Mantid. Remember the Presenter is not supposed to know what concrete models it's using. You're leaking dependency information into the presenter with this. Just call it `getObjectNames`.